scheduler/scheduler_dfs.go

```diff
@@ -198,10 +199,19 @@ func (s *syncClient) resolveResourcesDfs(ctx context.Context, table *schema.Tabl
 				wg.Wait()
 				return
 			}
+			tableSemVal, _ := s.scheduler.singleTableConcurrency.LoadOrStore(table.Name+"-"+client.ID(), semaphore.NewWeighted(s.scheduler.singleTableMaxConcurrency))
```

Comment:
maybe it should be performed before depth semaphore acquire?

Reply:
We can investigate and test that out in a followup PR once #1410 is merged.